recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
Reimar it is hard to follow errors on your local machine. If you commit
those tests we can see the logs in the CIs and debug it. In this particular
error, for example, I don't know if were your netCDF4 1.2.4 is from. Also,
the dependence chain is netCDF4 -> libnetcdf -> curl. That may give us a
clue to what needs fixing/updating as curl is at the bottom of the stackand
installing modern versions of any software will cause it to upgrade.


**********************************************************
Filipe P. A. Fernandes
Physical Oceanographer

Email: ocefpaf@gmail.com

http://ocefpaf.github.io/
**********************************************************

On Tue, Jul 12, 2016 at 12:53 PM, ReimarBauer <notifications@github.com>
wrote:

> In recipes/mss/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/912#discussion_r70467436>
> :
>
> > +    git_url: https://bitbucket.org/wxmetvis/mss.git
> > +    git_rev: 1.1.0
> > +
> > +build:
> > +    number: 0
> > +    script: python setup.py install --record record.txt
> > +    entry_points:
> > +      - mss = msui.mss_pyui:main
> > +
> > +requirements:
> > +  build:
> > +    - basemap ==1.0.7
> > +    - chameleon >=2.24
> > +    - curl ==7.26
> > +    - cycler ==0.10.0
> > +    - execnet ==1.2.0
>
> recent curl 7.4.9 needs netCDF1.2.4 and that changes to this issue
>
> import mss_wms_settings
> File "/home/mss/config/mss_wms_settings.py", line 56, in <module>
> import netCDF4
> File "/home/mss/miniconda2/lib/python2.7/site-packages/netCDF4/__init__.py", line 3, in <module>
> from ._netCDF4 import *
> libmfhdf.so.0: cannot open shared object file: No such file or directory
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/912/files/7b79a324cf340c1510293d57b43a00e02be51383#r70467436>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA6BL7N3V27EYg9cx_ceZktYpP0YT4gSks5qU7h_gaJpZM4JAyUH>
> .
>